src/Services/SubscriptionService.php

```diff
@@ -113,6 +113,14 @@ public function cancel($subscriptionId, $data)
         );
     }
 
+    /**
+     * @param string $subscriptionId
+     */
+    public function delete($subscriptionId)
+    {
+        return $this->client()->delete([], 'subscriptions/{subscriptionId}', ['subscriptionId' => $subscriptionId]);
```

Comment:
This doesn't look correct

```suggestion
        return $this->client()->delete('subscriptions/{subscriptionId}', ['subscriptionId' => $subscriptionId]);
```